doc/rtd/explanation/format.rst

```diff
@@ -3,18 +3,65 @@
 User data formats
 *****************
 
-User data is opaque configuration data provided by a platform to an instance at
-launch configure the instance. User data can be one of the following types.
+User data is configuration data provided by a user of a cloud platform to an
+instance at launch. User data can be passed to cloud-init in any of many
+formats documented here. Each format will document a required header or
+content-type that must be present in the user data to be recognized as that
+format.
+
+Configuration types
+===================
+
+User data can be categorized into **base config**
+or **meta config**.
+
+Base config
+-----------
+
+Any of the base configs will be used to directly configure the instance.
+These include:
+
+- `Cloud config data`_
+- `User data script`_
+- `Cloud boothook`_
+
+Meta config
+-----------
+
+Meta configs serve as a container, template, or means to obtain or modify
+a base config. These include
+
+- `MIME multi-part archive`_
+- `Cloud config archive`_
+- `Jinja template`_
+- `Include file`_
+- `Gzip compressed content`_
+- `Part handler`_
 
 .. _user_data_formats-cloud_config:
 
 Cloud config data
 =================
 
-Cloud-config is the preferred user data format. The cloud config format is a
-declarative syntax which uses `YAML version 1.1`_ with keys which describe
-desired instance state. Cloud-config can be used to define how an instance
-should be configured in a human-friendly format.
+| **Header:** #cloud-config
+| **Content-Type:** text/cloud-config
+
+Example
+-------
```

Comment:
I don't know that we need this or the explanation header to be subheaders. For consistency with the header and content type, would it make sense to set these to **Example** and **Explanation**? The sections don't feel long enough to justify having subsections in them.

Reply:
I think I prefer subheadings rather than bold, but I pushed a bold commit for comparison.

Reply:
It's nothing I'd be willing to hold up the doc over. If you're happier with the subheadings, we'll leave them as they are.